packages/server/src/api/controllers/table/utils.ts

```diff
@@ -139,8 +139,7 @@ export async function importToRows(
     const processed = await inputProcessing(user?._id, table, row, {
       noAutoRelationships: true,
     })
-    row = processed.row
-    table = processed.table
```

Comment:
What happened to the table here - I assume this is the part where the table is getting saved inside `inputProcessing` rather than being returned?

Reply:
Correct, we no longer need to "carry" the table through the loop.